CONTRIBUTING.md

```diff
@@ -28,18 +28,28 @@ Please clean up any messes that you find, and don't leave behind new messes for
 ## Contributing to the default libraries.
 
 Idris ships with a set of packages in `libs/` that is provided as a default library.
+
++ `prelude` is the bare minimum required for any Idris program to run.
++ `base` is tried and tested code that may be useful, and has seen active use in multiple projects.
++ `contrib` is code that is experimental in design and want to test before possible inclusion in `base`.
++ `effects` is an implementation of algebraic effects that is packaged alongside Idris.
+
 These packages should not be seen as the *standard* as when working with dependent types we do not necessarily know how best to work with dependent types.
 These packages offer functionality that can be built on top of when constructing Idris programs.
 
-One major point to make is that everything in prelude will be imported automatically, unless given
-the `--noprelude` option.
+One major point to make is that everything in prelude will be imported automatically, unless given the `--noprelude` option.
```

Comment:
Perhaps we should remove the bit "One major point to make is that "? It seems unnecessary to the meaning of the sentence.